gradle/docker.gradle

```diff
@@ -1,6 +1,6 @@
 jib {
     from {
-        image = "openjdk:16-jdk-alpine"
+        image = "eclipse-temurin:17-jdk-alpine"
```

Comment:
```suggestion
        image = "eclipse-temurin:17-jdk"
```

Reply:
Probably the whole gradle jib build doesn't even work but didn't test it yet. Just reused the alpine image therefore.

Reply:
I think the jib plugin is unused and does not produce working images. See #5740.

Reply:
Will not change this as proposed as it will be deleted either way in #5740.